lib/simple_sitemap/generators/index.rb

```diff
@@ -24,21 +25,25 @@ def add_sitemap(name)
       end
 
       def write!
-        xml = to_xml
-        index_filename = 'index.xml'
-        if @config.prefix
-          index_filename = "#{@config.prefix}_#{index_filename}"
+        @sitemaps.each_slice(SimpleSitemap::MAX_INDEX_SIZE) do |sitemaps|
+          xml = to_xml(sitemaps)
+          write_file index_filename, xml
+          @index_count += 1
```

Comment:
Could you use `with_index` instead of maintaining this index yourself? Otherwise :+1: